src/App.tsx

```diff
@@ -1,9 +1,17 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState<string>('');
+
+  const normalizeQuery = query.toLowerCase().trim();
+
+  const visibleMovies = moviesFromServer
+    .filter(movie => movie.title.toLowerCase().includes(normalizeQuery)
```

Comment:
DRY
make a separate function to check if str includes substr
also, split this with variables

Reply:
Fixed